cs3/sharing/collaboration/v1beta1/resources.proto

```diff
@@ -70,6 +70,9 @@ message Share {
   // Optional.
   // The expiration time of the share.
   cs3.types.v1beta1.Timestamp expiration = 9;
+  // Optional.
+  // Hide share, default false
+  bool hide = 10;
```

Comment:
```suggestion
  bool hidden = 10;
```

(Assuming we want to keep it, though it would rather belong to the `ReceivedShare` message.

Reply:
@dragonchaser I agree with @glpatcern ... I'd also use `hidden`. And the `hidden` status of share is used by recipients to hide shares they are not interested in. So the property should live in ReceivedShare. Otherwise the hidden state is shared by all recipients of the share. ... hm we can take a look at the implementation. AFAIR storing the property on the share made sense because the received share did not exist, yet. Or because listing shares should omit hidden shares?

Reply:
Yes this is on my list, will tackle this as soon as @aduffeck is done with the ocm story. I fear this is going to be a rebase hell for both of us otherwise :)